.gitignore

```diff
@@ -31,3 +34,5 @@ venv
 *.gdf
 *~
 *.iml
+pynestml/codegeneration/resources_spinnaker/common/iaf_psc_exp_template.c
```

Comment:
Should these two lines be here?

``.github`` should not be ignored, it contains the CI scripts.

I'm fine with your other changes in this file, but can you spin them off to a separate pull request directly to https://github.com/nest/nestml? Cheers!